packages/pools/src/utils.ts

```diff
@@ -0,0 +1,27 @@
+// should be defined somewhere "project" wide
+const REST_ENDPOINT = "https://lcd.osmotest5.osmosis.zone";
```

Comment:
https://github.com/osmosis-labs/osmosis-frontend/blob/b80b0235a6c3eafe3e9e0a8bf3436e7a553638c2/packages/web/server/queries/cosmos/bank/balances.ts#L16

Please reference this for getting the base URL to query from. Also, you may need to pass in the base URL from the transmuter pool since we only have access to chain configs from web package.

Reply:
Wasn't really sure how you guys wanted me to handle that so I left that comment in there.

Updated now as you suggested, thanks